lib/system/index.js

```diff
@@ -24,6 +25,7 @@ var capitalize = function(str) {
 module.exports    = system;
 system.os_name    = os_name;
 system.os_release = os.release();
+system.user_agent = 'Prey/' + version + ' (Node '+ process.version + ' ' + system.os_name[0].toUpperCase() + system.os_name.slice(1) + ' ' + system.os_release + ')';
```

Comment:
I don't think I quite get the comment. `slice` is a String function:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice